### PR TITLE
[WIP] Training ROR Test

### PR DESCRIPTION
**Pull request by hua-phuoc-mulodo**
Test post controller with actions: index, show, create, destroy. I have use support spec.

### CODE REVIEW

**Comment by hua-phuoc-mulodo** [comment]
I added action update to post controller test.

**Comment by hua-phuoc-mulodo** [comment]
[Training-test] - I pushed views test.

File: spec/controllers/posts_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        expect(@post.content).to eq("Post content 2")
+      end
+    
+      it "redirects to the updated" do
+        put :update, params: { id: @post, post: FactoryGirl.attributes_for(:post) }
+        expect(response).to redirect_to @post
+      end
+    end
+    
+    context "when attributes is valid invalid" do
```
Comment (tran-hai-mulodo) [inline spec/controllers/posts_controller_spec.rb]: @hua-phuoc-mulodo 
Please remove 'valid' word, it's confused! Thanks.

Comment (hua-phuoc-mulodo) [reply spec/controllers/posts_controller_spec.rb]: Ok thanks, I fixed.

**Comment by tran-hai-mulodo** [comment]
@hua-phuoc-mulodo 
It's OK. I merged. Thanks.

**Pull request merged**